corehq/apps/hqadmin/views.py

```diff
@@ -531,13 +531,15 @@ def get_context_data(self, **kwargs):
             num_cases = 0
             num_locations = 0
         formatted_payload = etree.tostring(xml_payload, pretty_print=True)
+        hide_xml = self.request.GET.get('hide_xml') == 'true'
```

Comment:
Is there any substantial work that can be skipped if the xml isn't needed?  I'm guessing most of it needs to be run for the timings to be accurate, huh?

Reply:
No, we actually need to get the restore for timing purposes